meshroom/ui/qml/Controls/SearchBar.qml

```diff
@@ -36,10 +58,46 @@ FocusScope {
             Layout.fillWidth: true
             selectByMouse: true
 
+            rightPadding: clear.width
+
+            // The text field is visible either when toggle is not activated or the visible property is set
+            visible: root.toggle ? root.isVisible : true
+
             // Ensure the field has focus when the text is modified
             onTextChanged: {
                 forceActiveFocus()
             }
+
+            // Handle enter Key press and forward it to the parent
+            Keys.onPressed: (event)=> {
```

Comment:
The core signal from the class is updated to be called `accepted` but using `onAccepted` instead of `Keys.onPressed` currently is emitting the signal twice so keeping it same at the moment, 
Have updated the event to consider both Return and Number Pad Enter key though.